onboarding-workflow_v5/src/triggers/upload_repos_from_queue.ts

```diff
@@ -21,7 +21,11 @@ async function uploadRepos() {
             console.log('Start upload of repo ', github.id)
             console.log('Repo url ', github.github_url)
             console.log('Number of objects ', github.objects)
-            await initializeGoshRepo(github.id)
+            try {
+                await initializeGoshRepo(github.id)
+            } catch (err) {
+                console.error('Failed to upload repo', err)
```

Comment:
I would make different messages in different error handlers, so it would be easier to trace where this message came from.
As a stretch you could separate try-catch into 2 blocks instead of nesting.